src/main/java/apoc/number/exact/Exact.java

```diff
@@ -0,0 +1,99 @@
+package apoc.number.exact;
+
+import apoc.Description;
+import org.apache.commons.lang.StringUtils;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.UserFunction;
+
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+
+import static java.lang.Math.pow;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 17 May 2017
+ */
+public class Exact {
+
+	@UserFunction
+	@Description("apoc.number.exact.add(stringA,stringB) - return the sum's result of two large numbers")
+	public String add(@Name("stringA")String stringA, @Name("stringB")String stringB){
+		if(stringA == null || stringA.isEmpty() || stringB == null || stringB.isEmpty()) return null;
+		return new BigDecimal(stringA).add(new BigDecimal(stringB)).toPlainString();
+	}
+
+	@UserFunction
+	@Description("apoc.number.exact.sub(stringA,stringB) - return the substraction's of two large numbers")
+	public String sub(@Name("stringA")String stringA, @Name("stringB")String stringB){
+		if(stringA == null || stringA.isEmpty() || stringB == null || stringB.isEmpty()) return null;
+		return new BigDecimal(stringA).subtract(new BigDecimal(stringB)).toPlainString();
+	}
+
+	@UserFunction
+	@Description("apoc.number.exact.mul(stringA,stringB,[prec],[roundingModel]) - return the multiplication's result of two large numbers ")
```

Comment:
if prec and roundingMode are optional, we should have defaultValue in their @Name annotations for all methods, that getMatchContext can react to sensibly. @AngeloBusato